### PR TITLE
Browser Back Button

### PR DESCRIPTION
**Pull request by ElvisJG**
browser back button useEffect, uses popstate to ensure window goes back

### CODE REVIEW

**Review by admoorgit — CHANGES_REQUESTED** [review]
It works! However, back button loads homepage to wherever scroll height was before leaving the page. Can the view be at the top of the page? Will be less disorienting.

**Comment by ElvisJG** [comment]
> It works! However, back button loads homepage to wherever scroll height was before leaving the page. Can the view be at the top of the page? Will be less disorienting.



> It works! However, back button loads homepage to wherever scroll height was before leaving the page. Can the view be at the top of the page? Will be less disorienting.

done 👍

File: src/pages/Search/DesktopResults.tsx [hunk 1]
```diff
@@ -29,6 +29,13 @@ function DesktopResults({ results }: { results: CareProviderSearchResult[] }) {
     rerenderMap();
   }, [results]);
 
+  useEffect(() => {
```
Comment (jlhogan) [inline src/pages/Search/DesktopResults.tsx:32]: what does this do? can we add a comment to the code about the what/why here?

Comment (ElvisJG) [reply src/pages/Search/DesktopResults.tsx:35]: done 👍

**Review by jlhogan — APPROVED** [review]

**Review by admoorgit — COMMENTED** [review]
Works as expected! I FORGOT TO SELECT APPROVED BUT I APPROVE OMG HOW DO I APPROVE NOW

**Review by admoorgit — APPROVED** [review]
Works as expected!

**Pull request merged**